docs/concepts/comparison.rst

```diff
@@ -33,19 +31,17 @@ Similar Libraries
 +-------------------------------------------+--------------------------+--------------------------------------+----------------------------+
 | Supports multiple environments / backends | Yes                      | Yes                                  | No                         |
 +-------------------------------------------+--------------------------+--------------------------------------+----------------------------+
-| Supports SourceBots Servo Board           | Yes                      | Yes                                  | No                         |
+| Supports SourceBots Servo Board           | Upcoming                 | Yes                                  | No                         |
 +-------------------------------------------+--------------------------+--------------------------------------+----------------------------+
 | Supports SR v4 Kit                        | Yes                      | Partial Support                      | Yes                        |
 +-------------------------------------------+--------------------------+--------------------------------------+----------------------------+
-| Supports vendor hardware modules          | Yes                      | No                                   | No                         |
-+-------------------------------------------+--------------------------+--------------------------------------+----------------------------+
 | Test Coverage                             | > 97%                    | Some on robot-api                    | No                         |
 +-------------------------------------------+--------------------------+--------------------------------------+----------------------------+
 | Type Checking                             | Yes                      | Partial                              | No                         |
```

Comment:
Type checking isn't available in Python 2
```suggestion
| Type Checking                             | Yes                      | Partial                              | N/A                         |
```

Reply:
Type annotations aren't available in Python 2. Typing can be achieved through type comments though

Reply:
It is, but uses pre-3.5 syntax: https://mypy.readthedocs.io/en/latest/python2.html